QUARKUS-1159.md

```diff
@@ -67,6 +67,14 @@ Strimzi Kafka running over SSL or using SASL authN is not working over FIPS
 
 Issue Ref: https://issues.redhat.com/browse/QUARKUS-1769
 
+### Non-FIPS Vs FIPS coverage
+
+- Date 29/07/2022
```

Comment:
```suggestion
Date: 2022-07-29
Tests: https://github.com/quarkus-qe/quarkus-test-suite `2.7` branch using RHBQ 2.7.6.GA artifacts
```
There is always concern with US and EU date form, the above one is more explicit for both continents.
Provide details about used TS and bits